homeassistant/components/pyload/config_flow.py

```
@@ -0,0 +1,154 @@
+        """Import config from yaml."""
+        # in config yaml all variables are optional, but some have a default value
+        config = {
+            CONF_NAME: import_info.get(CONF_NAME, DEFAULT_NAME),
+            CONF_HOST: import_info.get(CONF_HOST, DEFAULT_HOST),
+            CONF_PASSWORD: import_info.get(CONF_PASSWORD, ""),
+            CONF_PORT: import_info.get(CONF_PORT, DEFAULT_PORT),
+            CONF_SSL: import_info.get(CONF_SSL, False),
+            CONF_USERNAME: import_info.get(CONF_USERNAME, ""),
+            CONF_VERIFY_SSL: False,
+        }
+
+        result = await self.async_step_user(config)
+        # Raise an issue that this is deprecated and has been imported
+        if (
+            result.get("type") == FlowResultType.CREATE_ENTRY
+            or result.get("reason") == "already_configured"
+        ):
+            async_create_issue(
+                self.hass,
+                HOMEASSISTANT_DOMAIN,
+                f"deprecated_yaml_{DOMAIN}",
+                is_fixable=False,
+                issue_domain=DOMAIN,
+                breaks_in_ha_version="2025.2.0",
+                severity=IssueSeverity.WARNING,
+                translation_key="deprecated_yaml",
+                translation_placeholders={
+                    "domain": DOMAIN,
+                    "integration_title": "pyLoad",
+                },
+            )
+        elif errors := result.get("errors"):
+            error = errors["base"]
+
+            async_create_issue(
+                self.hass,
+                DOMAIN,
+                f"deprecated_yaml_import_issue_{error}",
+                breaks_in_ha_version="2025.2.0",
+                is_fixable=False,
+                issue_domain=DOMAIN,
+                severity=IssueSeverity.WARNING,
+                translation_key=f"deprecated_yaml_import_issue_{error}",
+                translation_placeholders=ISSUE_PLACEHOLDER,
+            )
```

Let's keep the issue creating in the place where we start the import, that makes it a bti cleaner imo

Also, when there is an error, we should abort instead of return an error